src/options.rs

```diff
@@ -764,6 +774,7 @@ impl Options {
         match builder.build_env().await {
             Ok(config) => {
                 let mut cfg = with_password(&self.conn_options, config).await?;
+                cfg = with_branch(&self.conn_options, cfg).await?;
```

Comment:
If I'm running `edgedb -I another_instance` in an EdgeDB project, it will pick up the wrong branch name.

Reply:
Fixed with the removal of `edgedb.auto.toml`